packages/sign-client/test/shared/init.ts

```diff
@@ -56,16 +56,14 @@ export async function initTwoPairedClients(
         initTwoClients(clientOptsA, clientOptsB, sharedClientOpts),
         TESTS_CONNECT_TIMEOUT,
       )) as Clients;
-
       const settled: any = await createExpiringPromise(
         testConnectMethod(clients),
-        TESTS_CONNECT_TIMEOUT,
+        TESTS_CONNECT_TIMEOUT * 2,
```

Comment:
Why `* 2`? Sorry if I missed something here, is something special with the tests?

Reply:
allows for several additional retries in case of flaky ci network